processor/pom.xml

```diff
@@ -145,6 +149,11 @@
                         </goals>
                         <configuration>
                             <minimizeJar>true</minimizeJar>
+                            <artifactSet>
+                                <excludes>
+                                    <exclude>${project.groupId}:mapstruct-spi</exclude>
```

Comment:
There certainly has been a reason why I did that, but I don't remember what it was. Having the SPI parts in the fat-jar is indeed convenient.

For maven builds with a custom SPI implementation, the SPI interfaces would be on the plugin classpath twice:
```
maven-processor-plugin -> custom-spi-impl -> mapstruct-spi
maven-processor-plugin -> mapstruct-processor -> mapstruct-spi
```
I'll check if that causes any issues, and remove the exclude if it doesn't.